### ISSUE TITLE
problems with git-server

### ISSUE THREAD

**Issue by tripodsan** [adobe/helix-simulator#92]
the git-server dependency is set to `"^0.9.7"` which includes node-git@0.27.2.
but since a newer git-server 0.9.10 is released, which includes node-git@0.23.0 downstream dependencies will use the later git-server.

in circleci, the newer nodegit leads to problems, but it is not possible to revert back to the old version of helix-simulator.

```
$ npm ls nodegit
@adobe/helix-cli@0.9.1-pre.0 /Users/tripod/codez/helix/project-helix/modules/helix-cli
└─┬ @adobe/petridish@2.5.0
  └─┬ @adobe/git-server@0.9.10
    └── nodegit@0.23.0

--> after addinh nodegit as hard dependency to helix-cli:

$ npm ls nodegit
@adobe/helix-cli@0.9.1-pre.0 /Users/tripod/codez/helix/project-helix/modules/helix-cli
└─┬ @adobe/git-server@0.9.7
  └── nodegit@0.22.2
```

### Suggest
* either use fixed reference for the `git-server` dependency here,
* or, git-server should use semver and increase at least the minor version when using a potential breaking change of nodegit.

**Comment by stefan-guggisberg** [comment]
i agree, +1 for using fixed version for the `git-server` dependency in `helix-simulator`.

**Comment by stefan-guggisberg** [comment]
resolved by using fixed version for git-server dependency (284ccda).
